contracts/interfaces/IPrizeTierHistoryV2.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.6;
+
+import "@pooltogether/v4-core/contracts/DrawBeacon.sol";
+
+/**
+ * @title  PoolTogether V4 IPrizeTierHistoryV2
+ * @author PoolTogether Inc Team
+ * @notice IPrizeTierHistoryV2 is the base contract for PrizeTierHistoryV2
+ */
+interface IPrizeTierHistoryV2 {
+    /**
+     * @notice PrizeTierV2 struct
+     * @dev    Adds Draw Percentage Rate (DPR) and minPickCost paramater to PrizeTier struct
+     */
+    struct PrizeTierV2 {
+        uint8 bitRangeSize;
+        uint32 drawId;
+        uint32 maxPicksPerUser;
+        uint32 expiryDuration;
+        uint32 endTimestampOffset;
+        uint32 dpr;
```

Comment:
How would we store a DPR of `0.02%`? We'll need to do some decimal shifting for this value.

Reply:
Yes, in decimal notation: `0.02% = 0.0002`. Then we multiply by 1e9, so `0.0002 * 1000000000 = 200000 = 2 * 1e5 = 0.02 * 1e7`
So we can shift the percentage number by `1e7`.